### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by NikolaevaIrina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
...
-4. Filter the target view in the event handler.
+2. In the `OnActivated` method, retrieve `DashboardViewItems` via the `FindItem` method. Also, subscribe to the `ControlCreated` event of a `DashboardViewItem` whose `ListView` is used to filter data (hereinafter referred to as `SourceView`).
+4. In the `ControlCreated` event handler retrieve the `SourceView` via the [DashboardViewItem.InnerView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Editors.DashboardViewItem.InnerView?p=netframework) property and subscribe to its [SelectionChanged](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectionChanged?p=netframework) event.
+5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same manner as in the previous step.
+6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. This object must be loaded from `SourceView` Object Space to `TargetView` Object Space via the `GetObject` method.
```
Comment (AndreyKozhevnikov) [inline Readme.md]: To get an object to filter on, use the <a href="https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.[View.SelectedObjects](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectedObjects)">View.SelectedObjects</a> property. (the second sentence can be removed)

File: Readme.md [hunk 2]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
...
+7. Now you can add `CriteriaOperator` to the [TargetView.CollectionSource.Criteria](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.CollectionSourceBase.Criteria?p=netframework) dictionary to filter the `TargetView`. In this example, we created a `InOperator` to filter the **Position** column via objects from `SourceView`.
```
Comment (AndreyKozhevnikov) [inline Readme.md]: to filter the 'AssignedTo' column via objects...

File: Readme.md (hunk 1 above)
Comment (NikolaevaIrina) [reply Readme.md]:
```suggestion
6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. 
```

File: Readme.md (hunk 2 above)
Comment (NikolaevaIrina) [reply Readme.md]:
```suggestion
7. Now you can add `CriteriaOperator` to the [TargetView.CollectionSource.Criteria](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.CollectionSourceBase.Criteria?p=netframework) dictionary to filter the `TargetView`. In this example, we created a `InOperator` to filter the **AssignedTo** column via objects from `SourceView`.
```

File: Readme.md [hunk 3]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
 
 ## Implementation Details
 1. Add a new [ViewController](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ViewController) to the **YourSolutionName.Module** project. For more information, refer to the following file: [DashboardFilterController.cs](./CS/EFCore/DependentDashboardEF/DependentDashboardEF.Module/Controllers/DashboardFilterController.cs).
-2. Access the target and source views.
-3. Handle the source view's list view selection changed event.
-4. Filter the target view in the event handler.
+2. In the `OnActivated` method, retrieve `DashboardViewItems` via the `FindItem` method. Also, subscribe to the `ControlCreated` event of a `DashboardViewItem` whose `ListView` is used to filter data (hereinafter referred to as `SourceView`).
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
2. In the `OnActivated` method, retrieve `DashboardViewItems` via the `FindItem` method, and subscribe to the `ControlCreated` event of a `DashboardViewItem` whose `ListView` is used to filter data (hereinafter referred to as `SourceView`).
```

File: Readme.md [hunk 4]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
...
+4. In the `ControlCreated` event handler retrieve the `SourceView` via the [DashboardViewItem.InnerView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Editors.DashboardViewItem.InnerView?p=netframework) property and subscribe to its [SelectionChanged](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectionChanged?p=netframework) event.
+5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same manner as in the previous step.
+6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property.
```
Comment (dirk-pieterse) [inline Readme.md]: `an object to filter on` Please explain what this means

File: Readme.md [hunk 5]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
...
+4. In the `ControlCreated` event handler retrieve the `SourceView` via the [DashboardViewItem.InnerView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Editors.DashboardViewItem.InnerView?p=netframework) property and subscribe to its [SelectionChanged](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectionChanged?p=netframework) event.
+5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same manner as in the previous step.
+6. To get an object to filter on, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. 
+7. Now you can add `CriteriaOperator` to the [TargetView.CollectionSource.Criteria](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.CollectionSourceBase.Criteria?p=netframework) dictionary to filter the `TargetView`. In this example, we created a `InOperator` to filter the **AssignedTo** column via objects from `SourceView`.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
7. You can now add `CriteriaOperator` to the [TargetView.CollectionSource.Criteria](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.CollectionSourceBase.Criteria?p=netframework) dictionary to filter the `TargetView`. In this example, we created an `InOperator` that uses `SourceView` objects to filter the **AssignedTo** column.
```

File: Readme.md [hunk 6]
```diff
@@ -16,9 +15,11 @@ When a [DashboardView](https://docs.devexpress.com/eXpressAppFramework/DevExpres
...
-2. Access the target and source views.
-3. Handle the source view's list view selection changed event.
-4. Filter the target view in the event handler.
+2. In the `OnActivated` method, retrieve `DashboardViewItems` via the `FindItem` method. Also, subscribe to the `ControlCreated` event of a `DashboardViewItem` whose `ListView` is used to filter data (hereinafter referred to as `SourceView`).
+4. In the `ControlCreated` event handler retrieve the `SourceView` via the [DashboardViewItem.InnerView](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.Editors.DashboardViewItem.InnerView?p=netframework) property and subscribe to its [SelectionChanged](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.View.SelectionChanged?p=netframework) event.
+5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same manner as in the previous step.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
5. In the `SelectionChanged` event handler, retrieve the View to be filtered (hereinafter referred to as `TargetView`) in the same way as the previous step.
```

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 4 above)
Comment (NikolaevaIrina) [reply Readme.md]: Hi Dirk. I was trying to say, `to get an object that is used for filtering`. Can we rephrase this sentence in the following way?

```suggestion
6. To get an object that is used for filtering, use the [ListView.CurrentObject](https://docs.devexpress.com/eXpressAppFramework/DevExpress.ExpressApp.ListView.CurrentObject?p=netframework) property. 
```

Comment (dirk-pieterse) [reply Readme.md]: Reads better now, thanks

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**